statistics/statistics.py

```diff
@@ -10,6 +10,27 @@
 from qstrader import settings
 
 class Statistics():
+    
+    def __init__(self, portfolio_handler):
+        self.portf_hand = portfolio_handler
+
+    def update(self, event_time):
+        """On every BAR or TICK event that update() is called, unrealised 
+        pnl is written to the equity.csv file in the output folder"""
+        unrealized_pnls_snapshot = []
+        for ticker in self.portf_hand.portfolio.positions:
+            pt = self.portf_hand.portfolio.positions[ticker]
+            unrealized_pnls_snapshot.append(float(pt.unrealised_pnl))
```

Comment:
Do we want to use decimal instead of float here?
(https://docs.python.org/2/library/decimal.html)